projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/ai-assistant/with-ai-assistant.tsx

```diff
@@ -120,20 +133,20 @@ export const withAIAssistant = createHigherOrderComponent(
 			);
 
 			removeBlocks( otherBlocksIds );
-		}, [ blocks, blockType, content, replaceBlock, clientIds, removeBlocks ] );
+		}, [ blockType, replaceBlock, removeBlocks ] );
 
 		const blockControlProps = {
 			group: 'block',
 		};
-		const rawContent = getRawTextFromHTML( content );
+		// const rawContent = getRawTextFromHTML( content );
```

Comment:
You should move any attempt to disable `AiAssistantDropdown` into `AiAssistantDropdown`. It's wrapped in `BlockControls` so it runs much less often, only when the block controls are displayed.